.github/docs_pr_change_template.md

```diff
@@ -0,0 +1,10 @@
+## Description
+
+<!-- Describe the changes made in this PR -->
+
+## Checklist
+
+- [ ] This change matches the [docs style guide](https://goteleport.com/docs/contributing/documentation/style-guide/), and the terms used in this change correspond to those in [Teleport Core Concepts](https://goteleport.com/docs/core-concepts)
+- [ ] I viewed the changes on the dev server (`yarn dev`) to confirm they render without error
```

Comment:
Are there good instructions on how to do this? 

Confession, I've merged dozens (if not more) docs PRs and I've never done this 😇

Reply:
How about:

```suggestion
- [ ] I viewed the changes on the [dev server](https://github.com/gravitational/docs/blob/main/README.md#getting-started) to confirm they render without error
```

Reply:
Perfect!